modules/apps/frontend-js/frontend-js-a11y-web/src/main/java/com/liferay/frontend/js/a11y/web/internal/configuration/FlagA11yConfiguration.java

```diff
@@ -12,4 +12,22 @@
  * details.
  */
 
-export {A11y} from './A11y';
+package com.liferay.frontend.js.a11y.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Matuzalem Teles
+ */
+@ExtendedObjectClassDefinition(generateUI = false)
+@Meta.OCD(
+	id = "com.liferay.frontend.js.a11y.web.internal.configuration.FlagA11yConfiguration"
+)
+public @interface FlagA11yConfiguration {
+
+	@Meta.AD(deflt = "true", required = false)
```

Comment:
@dsanz I'm thinking about sending this disabled for the time being, because we still need the Panel that didn't come in, and also maybe we have to work on an adoption strategy? I'm just worried that one day developers will wake up and see products with too many accessibility errors, some products have more, some less, some are our own components that teams must configure or we can support.

Reply:
I agree. I would officially release the a11y checker once side panel is ready, not before.

In addition, I think we should give ourselves (FI team) some time to test this in the portal. Reasons are:
- As this tool is targeted to developers, we as FI folks should get familiar with how tool works, what can we do with the info it gives, etc. This is a way to check how useful is what we created and get early feedback from target users.
- We'd like to see what's the perceived impact in performance when navigating in presence of the tool, particularly, in complex pages. This may lead to tweak the configurations or even make some deeper optimizations in the tool.
- We may find that some of the rule violations are caused by our own components. It'd be nice if we make sure no clay component or theme we own brings these issues, before tool is enabled by default. This would save valuable time and frustration to other product team devs.

I believe this will pay off, however, as these things require some effort (i.e. time) from our people, I'd like to agree on this plan with @nhpatt and @david-truong What do you guys think?

Reply:
I totally agree, I think it makes a lot of sense to experiment first before releasing it to everyone and fix the issues with our components, both on the Clay and DXP sides.

So for now I will keep this disabled before I go.